src/Commands/UserCommands/CancelCommand.php

```diff
@@ -54,15 +60,24 @@ public function execute()
      */
     public function executeNoDB()
     {
-        //Database not setted or without connection
-        $message = $this->getMessage();
-        $chat_id = $message->getChat()->getId();
+        //Output the default message when there is no DB connection
+        parent::executeNoDB();
```

Comment:
If you build a command with a keyboard you can have the possibility to clear the chat. 
Know that DB is not connected is excessive. Is just to let this command be as generic as possible!